src/app/src/components/NavBar.js

```diff
@@ -88,9 +88,7 @@ function ExitButton({ variant }) {
         </Button>
     ) : (
         <Button
-            onClick={() => {
-                navigate('/submissions');
-            }}
+            onClick={() => navigate(-1)}
```

Comment:
We noticed during RRP that this might navigate back to Welcome.

Reply:
[0df6d44](https://github.com/azavea/iow-boundary-tool/pull/156/commits/0df6d44c3c2c99f502bc40c72500760ffec13c91)